server/base/src/main/java/org/apache/accumulo/server/ServerConstants.java

```diff
@@ -68,7 +72,7 @@
   public static final int LOGGING_TO_HDFS = 4;
   public static final BitSet CAN_UPGRADE = new BitSet();
   static {
-    for (int i : new int[] {DATA_VERSION, MOVE_TO_ROOT_TABLE, MOVE_DELETE_MARKERS, LOGGING_TO_HDFS}) {
+    for (int i : new int[] {DATA_VERSION, MOVE_TO_REPLICATION_TABLE, MOVE_TO_ROOT_TABLE, MOVE_DELETE_MARKERS, LOGGING_TO_HDFS}) {
```

Comment:
bq. My gut reaction is to not worry about 1.5, but is there any fundamental problem in supporting that (is it just a testing burden)?

Yeah its just a matter of testing it.  I am not completely sure, but it seems code may be in place to support that upgrade. I don't know if it works.   Personally I would not want to support it w/o testing it.

bq. If we don't support a direct upgrade from 1.5, what's the fail-safe? 

Could add support for it in 1.8.1 or later if someone really wants that functionality.

> Could users just bulk-import the old files into a new instance?

That's tricky, if done incorrectly can resurrect old and/or deleted data.

Also before making any definitive decisions about 1.5 and earlier, we should see if the 1.6 to 1.8 upgrade uncovers any interesting issues.